examples/exec-outputs/cluster.sh

```diff
@@ -68,5 +68,12 @@ uninstall() {
   echo 'Uninstalling Cluster...'
 }
 
+assert-admin() {
```

Comment:
Instead of putting the porter integration test in the bundle, which isn't how we normally do example bundles, how about updating this test and put in any asserts that you want?

https://github.com/deislabs/porter/blob/1cff9e4713002cec674ee5aad516ad97832825aa/tests/outputs_test.go#L98

That way the example continues to demonstrate bundle techniques to people and the tests reside all in the same test file.

Reply:
Good point.  As we discussed offline, we still can't currently inspect docker driver output in the int tests, so moving the modifications to the bundle in `tests/testdata` (as opposed to an example bundle in `examples`) seemed to be a good compromise.